ButterflyMX Demo/Controllers/AccountTableViewController.swift

```diff
@@ -41,24 +41,40 @@ class AccountTableViewController: UITableViewController {
         if indexPath.section == 1 && indexPath.row == 0 {
             SVProgressHUD.show()
             
-            APIClient.shared.unRegisterDevice { result in
-                switch result {
-                case .success:
-                    // Will change this line when releasing a new SDK since I removed completion from logoutUser
-                    BMXCoreKit.shared.logoutUser{ result in }
-                    
-                    SVProgressHUD.dismiss()
-                    
-                    let stb = UIStoryboard(name: "Main", bundle: nil)
-                    let loginViewController = stb.instantiateViewController(withIdentifier: "LoginViewController")
-                    loginViewController.modalPresentationStyle = .fullScreen
-                    self.present(loginViewController, animated: true, completion: nil)
+            let tenants = BMXUser.shared.getTenants()
+            
+            let dispatchGroup = DispatchGroup()
+            
+            for tenant in tenants {
+                let key = "webhook-\(tenant.id)"
+                if let webhookId = UserDefaults.standard.string(forKey: key) {
```

Comment:
It's will not notify main queue if tenants will be empty

Reply:
If tenants are empty or webhookId is empty, the corresponding dispatchGroup.enter() won't be called,
but dispatchGroup.notify(queue: .main) will still be called.